core/shared/src/main/scala/zio/Exit.scala

```diff
@@ -77,7 +77,17 @@ sealed trait Exit[+E, +A] extends Product with Serializable { self =>
     }
 
   /**
-   * Effectfully folds over the value or cause.
+   * Replaces the value with the one provided.
+   */
+  final def const[B](b: B): Exit[E, B] = map(_ => b)
```

Comment:
Sorry to be the party pooper, but could you put that in alphabetical order? 🙏

Reply:
yes! was just doing that. thanks :-)

Reply:
merci ✌️